Makefile

```diff
@@ -203,10 +205,31 @@ generate-fakes: ## Generate fakes for writing unit tests
 	$(GO) generate ./...
 	$(MAKE) fmt
 
+.PHONY: e2e-cli-core
+e2e-cli-core: lint tools ## Run CLI Core E2E Tests
```

Comment:
I forget, why do we choose to run "lint" here? It means or runs twice in CI which slows it down.

Reply:
i don't it being ran as part of `build`

Reply:
I have removed it here.

Reply:
The linter is not run in "make build" but it's run in the other CI workflow.